src/environs/__init__.py

```diff
@@ -463,6 +463,7 @@ def read_env(
         False as a second positional argument.
         """
 
+        env_path = ""
```

Comment:
This results in an empty string being returned if a `.env` file isn't found. Not sure if this is the most intuitive; perhaps `None` should be returned?

Reply:
`None` works as well but my thought was to mimic the same behavior as `find_dotenv()`. None and an empty string works the same way for conditional checks, but I think None is more representative indeed.